src/ai/backend/agent/docker/agent.py

```diff
@@ -921,6 +932,11 @@ def _mount(type: MountTypes,
                 'seccomp=unconfined',
                 'apparmor=unconfined',
             ]
+        ports = ''
+        for port_no in preopen_ports:
+            ports += "," + str(port_no) + ':preopen:' + str(port_no)
+        container_config['Labels']['ai.backend.service-ports'] = \
+                image_labels['ai.backend.service-ports'] + ports
```

Comment:
Maybe we could write in more Pythonic way:
```suggestion
        encoded_preopen_ports = ','.join(f'{port_no}:preopen:{port_no}' for port_no in preopen_ports)
        container_config['Labels']['ai.backend.service-ports'] = \
                image_labels['ai.backend.service-ports'] + ',' + encoded_preopen_ports
```